simple_robotics_cpp_utils/build.sh

```diff
@@ -12,7 +12,8 @@ BUILD_PATH="$THIS_FILE_DIR/build"
 if [ -d $BUILD_PATH ]; then rm -rf $BUILD_PATH; fi
 mkdir $BUILD_PATH
 cd $BUILD_PATH
-cmake ..
```

Comment:
We decided to use CUSTOM_INSTALL_PATH for non-catkin packages. That is deemed as a good practice.